package.json

```diff
@@ -34,6 +34,7 @@
     "lodash": "^4.5.1",
     "mocha": "^2.4.5",
     "mustache": "^2.2.1",
+    "npm-run-all": "^4.0.2",
```

Comment:
Hi @m1kola, I have regenerated the `npm-shrinkwrap` file for you. Would be great if this can be tested on Windows and Linux again. Have tested it on my OSX machine.